### PR TITLE
Added a CommitteeFeedMessage entity

### PR DESCRIPTION
**Pull request by HeahDude**
Closes [EM-46](http://jira.sensiolabs.net/browse/EM-46).

### CODE REVIEW

**Comment by HeahDude** [comment]
Until #174 and #184 review only last commit e328b03.

File: app/Resources/views/committee/show.html.twig [hunk 1]
```diff
@@ -5,10 +5,93 @@
...
             <h2 id="committee-name">{{ committee.name }}</h2>
-            <h3 id="committee-description">{{ committee.description }}</h3>
+        </div>
+
+        <div>
+            {% if not is_granted('ROLE_ADHERENT') %}
+                <div class="committee-link committee-follow-disabled">
+                    Suivre ce comité
+                </div>
+                <a href="{{ path('app_membership_register') }}" id="committee-register-link">Adhérer à En Marche!</a>
```
Comment (romain-pierre) [inline app/Resources/views/committee/show.html.twig]: La typo officielle prend un espace avant le `!`

File: app/Resources/views/committee/show.html.twig [hunk 2]
```diff
@@ -5,10 +5,93 @@
...
+            <h5>Localisation</h5>
+            {#
+                TODO add static image
+            #}
+        </div>
+        <div class="committee-details">
+            <h5>Partager</h5>
+            <a href="{{ committee.facebookPageUrl }}">Facebook</a>
+            <a href="{{ committee.googlePlusPageUrl }}">Google +</a>
+            <a href="https://twitter.com/{{ committee.twitterNickname }}">Twitter</a>
```
Comment (romain-pierre) [inline app/Resources/views/committee/show.html.twig]: Are you sure that the nickname is checked before (Twitter's requirements about the format) ? In other case, may require an URL encoding.

File: src/AppBundle/Committee/CommitteeFeedHandler.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+class CommitteeFeedHandler
+{
+    private $manager;
+
+    public function __construct(ManagerRegistry $doctrine)
+    {
+        $this->manager = $doctrine->getManager();
+    }
+
+    public function createMessage(Feed\CommitteeMessage $message, Committee $committee, Adherent $adherent): CommitteeFeedMessage
```
Comment (hhamon) [inline src/AppBundle/Committee/CommitteeFeedHandler.php]: Namespace to be put on top.

File: src/AppBundle/Committee/Feed/CommitteeMessage.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace AppBundle\Committee\Feed;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class CommitteeMessage
+{
+    /**
+     * @Assert\NotBlank()
```
Comment (hhamon) [inline src/AppBundle/Committee/Feed/CommitteeMessage.php]: Extra parenthesis must be removed.

File: src/AppBundle/Twig/CommitteeExtension.php [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        return $this->authorizationChecker->isGranted(CommitteePermissions::CREATE, $committee);
+    }
+
+    public function canSee(Committee $committee): bool
+    {
+        return $this->authorizationChecker->isGranted(CommitteePermissions::SHOW, $committee);
+    }
+
+    public function getPath(string $routeName, Committee $committee): bool
```
Comment (hhamon) [inline src/AppBundle/Twig/CommitteeExtension.php]: Wrong return type: /bool/string/

File: src/AppBundle/Twig/CommitteeExtension.php [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        return $this->urlGenerator->generate($routeName, [
+            'uuid' => $committee->getUuid()->toString(),
+            'slug' => $committee->getSlug(),
+        ]);
+    }
+
+    public function getUrl(string $routeName, Committee $committee): bool
```
Comment (hhamon) [inline src/AppBundle/Twig/CommitteeExtension.php]: Wrong return type: /bool/string/

File: src/AppBundle/Committee/CommitteeFeedHandler.php (hunk 3 above)
Comment (HeahDude) [inline src/AppBundle/Committee/CommitteeFeedHandler.php]: This is a sub namespace, not need for a use statement.

File: src/AppBundle/Committee/Feed/CommitteeMessage.php (hunk 4 above)
Comment (HeahDude) [inline src/AppBundle/Committee/Feed/CommitteeMessage.php]: Fixed.

File: src/AppBundle/Twig/CommitteeExtension.php (hunk 5 above)
Comment (HeahDude) [inline src/AppBundle/Twig/CommitteeExtension.php]: Fixed in previous PR.

File: src/AppBundle/Twig/CommitteeExtension.php (hunk 6 above)
Comment (HeahDude) [inline src/AppBundle/Twig/CommitteeExtension.php]: Fixed in previous PR.

File: src/AppBundle/Committee/CommitteeFeedHandler.php [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Committee;
+use AppBundle\Entity\CommitteeFeedMessage;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+class CommitteeFeedHandler
+{
+    private $manager;
+
+    public function __construct(ManagerRegistry $doctrine)
```
Comment (hhamon) [inline src/AppBundle/Committee/CommitteeFeedHandler.php]: ObjectManager instead of ManagerRegistry.

File: app/config/services/committee.xml [hunk 8]
```diff
@@ -36,6 +36,10 @@
             <tag name="kernel.event_subscriber"/>
         </service>
 
+        <service id="app.committee.committee_feed_handler" class="AppBundle\Committee\CommitteeFeedHandler">
+            <argument type="service" id="doctrine" />
```
Comment (hhamon) [inline app/config/services/committee.xml]: Inject `doctrine.orm.entity_manager` instead.

File: src/AppBundle/Committee/CommitteeFeedHandler.php [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        $this->manager = $doctrine->getManager();
```
Comment (hhamon) [inline src/AppBundle/Committee/CommitteeFeedHandler.php]: No need for that when you inject an ObjectManager.

Comment (HeahDude) [inline src/AppBundle/Committee/CommitteeFeedHandler.php]: Indeed, fixed.

File: app/config/services/committee.xml (hunk 8 above)
Comment (HeahDude) [inline app/config/services/committee.xml]: Fixed.

File: src/AppBundle/Committee/CommitteeFeedHandler.php (hunk 7 above)
Comment (HeahDude) [inline src/AppBundle/Committee/CommitteeFeedHandler.php]: Fixed.

**Pull request merged**